readthedocs/projects/models.py

```diff
@@ -1233,8 +1233,7 @@ def show_advertising(self):
 
         return True
 
-    @property
-    def environment_variables(self):
+    def environment_variables(self, *, all=True):
```

Comment:
I would use `False` as default value, which I think it's safer.

Also, I may use a different name than `all`. Maybe `only_public=True` to be explicit about what's returning

Reply:
OK! Since `only_public` means flipping the meaning, flipping the default as well means it stays `True`

Reply:
If we use “return only publy” as a default, then the API would change behaviors, as the serializer only supports calling methods with their default parameters.